.github/workflows/ci-base.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Set up golang
       uses: actions/setup-go@v2
       with:
-        go-version: 1.19
+        go-version: 1.20.14
```

Comment:
```suggestion
        go-version-file: go.mod
```

Reply:
Oh, and update to `actions/setup-go@v5` when we're at it

Reply:
fair enough! let me split in two changes